### PR TITLE
refactor: avoid separate bash files for each new token being used as …

### PR DESCRIPTION
**Pull request by gbalabasquer**
…collateral (SC-7525)

### CODE REVIEW

**Comment by gbalabasquer** [comment]
Still need to do deeper review and testing, but it should be done.

**Comment by gbalabasquer** [comment]
Made some more reviewing and a full deployment to kovan and all looking good.
I think this is ready for review.

File: libexec/setters/set-ilks-line [hunk 1]
```diff
@@ -43,4 +43,3 @@ for token in $tokens; do
         setLine "${token}-${ilk}" "$(jq -r ".tokens.${token}.ilks.${ilk} | .line" "$CONFIG_FILE")"
     done
 done
```
Comment (lucas-manuel) [inline libexec/setters/set-ilks-line:45]: Dont we need to keep this newline?

Comment (gbalabasquer) [reply libexec/setters/set-ilks-line:45]: There were 2. When you do not leave a final newline Github will show some red icon in the diff.

**Review by brianmcmichael — APPROVED** [review]
LGTM. We'll need to update the collateral onboarding doc to reference these changes.

**Review by godsflaw — APPROVED** [review]
LGTM, same comment about the COB docs

**Pull request merged**